.deepsource.toml

```diff
@@ -34,3 +34,5 @@ enabled = true
 [[analyzers]]
 name = "secrets"
 enabled = true
+
+
```

Comment:
Can you please remove these additional newlines?

Reply:
I will see to it. Thank you